@here/harp-mapview/test/CopyrightProvidersTest.ts

```diff
@@ -18,35 +18,50 @@ import { UrlCopyrightProvider } from "../lib/copyrights/UrlCopyrightProvider";
 
 describe("CopyrightProviders", function() {
     describe("#UrlCopyrightProvider", function() {
+        function getCopyrightProvider() {
+            return new UrlCopyrightProvider("", "normal");
+        }
         async function getCopyrights(geoBox: GeoBox, level: number): Promise<CopyrightInfo[]> {
-            const provider = new UrlCopyrightProvider("", "normal");
+            const provider = getCopyrightProvider();
             return provider.getCopyrights(geoBox, level);
         }
 
         describe("#init", function() {
+            // tslint:disable: no-unused-expression
             it("Should return default copyrights if failed to load data", async function() {
-                const fakeJson = sinon.fake.rejects(new Error("error"));
+                const jsonerror = "jsonerror";
+                const fakeJson = sinon.fake.rejects(new Error(jsonerror));
 
                 sinon.replace(TransferManager.prototype, "downloadJson", fakeJson);
-                const copyrights = await getCopyrights(
+                const copyrightProvider = getCopyrightProvider();
+                const errorSpy = sinon.spy();
```

Comment:
this spy checks that the error message is logged, it is checking that the function logger.error is called in UrlCopyrightProvider line 43.